engine/org.eclipse.birt.report.engine.emitter.wpml/src/org/eclipse/birt/report/engine/emitter/wpml/writer/AbstractWordXmlWriter.java

```diff
@@ -50,6 +50,8 @@ public abstract class AbstractWordXmlWriter {
 
 	public static final int INDEX_NOTFOUND = -1;
 
+	protected final char SHY_CHAR = (char) 173;
```

Comment:
It took me a while to understand that SHY is SOFT_HYPHEN. Can you be more explicit about the name? You can remove CHAR from the name. Maybe you can also be more explicit in the declaration:

  protected final char SOFT_HYPHEN  = "\u00AD".charAt(0); // did not test this

Reply:
If this is the correct and accepted term then it is fine with me.